content/v2.0/reference/key-concepts/table-structure.md

```diff
@@ -17,6 +16,7 @@ InfluxDB 2.0 uses the following columnar table structure to store data:
 - **Header row:** describes the data labels for each column in a row.
 - **Data columns:** include the following columns: annotation, result, and table.
 - **Data rows:** all rows that contain time series data. For details about the type of data stored in InfluxDB, see [InfluxDB data elements](/v2.0/reference/key-concepts/data-elements/).
+- **Group keys** modify output tables in Flux by grouping records that share common values in specified columns. Learn more about [grouping your data with Flux](/v2.0/query-data/guides/group-data/).
```

Comment:
**Group keys** ~modify~ determine the contents of output tables...

This is a minor nit, so feel free to ignore. "Modify" feels very active, like it's something they're continually doing. Group keys are the result of user input and only change when acted upon. Like I said, really minor, feel free to ignore.